proxy/http/HttpSM.cc

```diff
@@ -200,7 +200,8 @@ HttpVCTable::cleanup_entry(HttpVCTableEntry *e)
     // Update stats
     switch (e->vc_type) {
     case HTTP_UA_VC:
-      //              HTTP_DECREMENT_DYN_STAT(http_current_client_transactions_stat);
+      // Moved to HttpSM::destroy
+      // HTTP_DECREMENT_DYN_STAT(http_current_client_transactions_stat);
```

Comment:
I don't know if we need to keep these commented out, but maybe  just a comment saying "Metrics are updated in ::destroy".